massgov/mayflower
# DP-9831 - Add contact and related links in sidebar to curated lists

## Description

Adds the sidebar to curated lists using a new custom stacked row template variation with the external sidebar (stacked-rows-with-external-sidebar-template.twig).

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-9831)

## Steps to Test

- [ ] Go to http://localhost:3000/?p=pages-listing-links.
- [ ] You should see the sidebar with examples of contact and related links:
![image](https://user-images.githubusercontent.com/1103479/43851268-2a7871f4-9b08-11e8-9a6f-630b404694e3.png)
- Scroll down. You will see the "Contact" section with the same sidebar contact info, but in accordion-style:
![image](https://user-images.githubusercontent.com/1103479/43925313-ef87e014-9bf4-11e8-904d-ad758c9138a3.png)

- [ ] View the MF page in mobile. You should see the contact accordion and related links below the main content.
- [ ] You should also see the sticky TOC on both desktop and mobile.
